assembl/static2/js/app/components/svg/circle.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { calculatePercentage } from '../../utils/globalFunctions';
+
+class Circle extends React.Component {
```

Comment:
This is actually a doughnut. I'm not kidding. This is the name you have in some graph libraries.